app/models/answer.rb

```diff
@@ -26,17 +26,17 @@ def likers
 
   def answer_notify(current_user)
     # notify question author
-    if current_user.uid != self.author.uid
-      AnswerMailer.notify_question_author(self.author, self).deliver
+    if current_user.uid != self.node.author.uid
```

Comment:
Here I'm not sure this is correct, @ananyo2012, can you take a look? I think we do want to notify original node (question) author when an answer is posted, but not the current user (that is, the author of the answer)